vexim/config/header.php

```diff
@@ -1,24 +1,25 @@
 <?php
-  if (isset($_SESSION['domain_id'])) {
-    $domheaderquery = "SELECT enabled FROM domains WHERE domains.domain_id=:domain_id";
-    $domheaderresult = $dbh->prepare($domheaderquery);
-    $domsuccess = $domheaderresult->execute(array(':domain_id'=>$_SESSION['domain_id']));
-    if ($domsuccess) { $domheaderrow = $domheaderresult->fetch(); }
-    $usrheaderquery = "SELECT enabled FROM users WHERE localpart=:localpart AND domain_id=:domain_id";
-    $usrheaderresult = $dbh->prepare($usrheaderquery);
-    $usrsuccess = $usrheaderresult->execute(array(':localpart'=>$_SESSION['localpart'], ':domain_id'=>$_SESSION['domain_id']));
-    if ($usrsuccess) { $usrheaderrow = $usrheaderresult->fetch(); }
-  }
-
-  print "<div id=\"Header\"><p><a href=\"https://github.com/avleen/vexim2\" target=\"_blank\">" . _("Virtual Exim") . "</a> ";
+  print "<div id=\"Header\"><p><a href=\"https://github.com/vexim/vexim2\" target=\"_blank\">" . _("Virtual Exim") . "</a> ";
```

Comment:
Why did you put output above `header()` calls?  I don't see why it would matter for the application logic, so it looks like the lines are reshuffled just for the sake of it. Please don't do this.